packages/react-core/src/components/Dropdown/Dropdown.tsx

```diff
@@ -114,20 +114,18 @@ const DropdownBase: React.FunctionComponent<DropdownProps> = ({
       ) {
         if (onOpenChangeKeys.includes(event.key)) {
           onOpenChange(false);
-          toggleRef.current?.focus();
+          toggleRef.current?.focus({preventScroll:true});
```

Comment:
I was still seeing the described problem on menudropdowns on filter views without the above change. 
Why this is being triggered on multi-select components I don't know. 
With the other fix in place, and this one not. 
The same behaviour below was occurring:

https://github.com/user-attachments/assets/f29b5fde-5d15-4ae8-8887-e322a14d0af0

Reply:
[This](https://github.com/content-services/content-sources-frontend/blob/a88fae033f857668c472a7cfe3e707f2a809b05e/src/Pages/Repositories/ContentListTable/components/SnapshotDetailsModal/Tabs/SnapshotErrataFilters.tsx#L166) is the above component.

[This](https://github.com/content-services/content-sources-frontend/blob/main/src/components/DropdownSelect/DropdownSelect.tsx#L28) is the wrapper I've built to contain the madness that is the patternfly Select component. 😆

Reply:
So I tried mimicking the setup from both files linked above just within a PatternFly example (wasn't able to get the repo linked in the above files running locally), and I'm not able to reproduce the jump scroll when simply removing the `preventScroll` option from this toggleRef.focus call of the Select component. May not be an exact 1:1 mimic, and it could also be a difference between the PatternFly workspace and yours, but as far as I can tell reverting this specific line in the changed files shouldn't cause that jump scroll bug when the menu is opened.

Personally I would opt to remove the options being passed to the `toggleRef.focus()` calls in these files (but keep the option in the other focus calls, i.e. `firstElement...focus()`), as the expected behavior should be that the page will scroll back to where the MenuToggle receives focus upon the Menu being closed via keyboard. Then if the jump scroll bug is still occurring with the rest of these updates merged in, we could revisit again to investigate further or add the options in as you did originally here.

How did you go about testing the updates in this PR in your product running locally? And you can confirm that reverting the `toggleRef.current?.focus({preventScroll:true});` lines in this PR back to just `toggleRef.current?.focus();` (but keeping all other changes) causes the bug to still occur when clicking on a MenuToggle to open the select/dropdown/menu?

Reply:
>>How did you go about testing the updates 
The other component changes were not tested; the copied logic matched and thus I added the changes as I presumed they would be desired. 

>>How did you go about testing the updates in this PR
I simply made the changes to the patternfly components in node_modules > patternfly, and rebuilt. 

>> Personally I would opt to remove the options being passed to the toggleRef.focus()
This is fine, I'll either use another component or find another way to prevent the behaviour.

>>can confirm that reverting the toggleRef.current?.focus({preventScroll:true})
I'll test again today

Reply:
Hey @Andrewgdewar , just wanted to check in on if you were able to test the above?

Also wanted to clarify in case I might be misinterpreting the issue here. Can you confirm between the foilowing which you originally meant in regards to this `toggleRef...focus` call needing the preventScroll option?

1. When the toggleRef focus call does NOT have the preventScroll option passed in, the jump scroll bug still occurs when the menu/select/drodown is opened
2. Or, were you saying that you believe that we shouldn't scroll the page when the menu/select/dropdown is closed and the MenuToggle (toggleRef in this case) receives focus?

Reply:
> When the toggleRef focus call does NOT have the preventScroll option passed in, the jump scroll bug still occurs when the menu/select/drodown is opened